tools/slang-test/main.cpp

```diff
@@ -691,7 +400,7 @@ OSError spawnAndWait(TestContext* context, const String& testPath, OSProcessSpaw
     if (err != kOSError_None)
     {
 //        fprintf(stderr, "failed to run test '%S'\n", testPath.ToWString());
-        context->messageFormat(TestMessageType::RunError, "failed to run test '%S'", testPath.ToWString().begin());
+        context->reporter->messageFormat(TestMessageType::RunError, "failed to run test '%S'", testPath.ToWString().begin());
```

Comment:
I suppose these diffs could be avoided if `TestContext` provided a helper `messageFormat` that forward to the real one, but the varargs would make that more complicated than is desirable.